PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -326,6 +345,29 @@ then
     log_Err_Abort "--printcom is not consistently implemented, do not rely on it"
 fi
 
+# Convert the --runmode string argument into a numeric code
+case "$RunMode" in
+    Default)
```

Comment:
Allow empty string as another way to get default behavior
```suggestion
    Default|"")
```